specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Note that `ballot_manifest_info` currently does not have `ballot_id` in it, but it should, right?